first.json

```diff
@@ -1 +1 @@
-{"name": "first", "learning_rate": 0.01, "epochs": 1, "batch_size": 1000, "val_acc": 0.9783}
```

Comment:
Generally, you probably wouldn't want to commit these raw outputs to the git repository. They can get quite big, especially the models (see below) and it will eventually kill your repo.

The way to fix this is to add the results directory to the .gitignore file. This does mean that you need to be careful about backing up your results (the ones you want to keep at least) but git doesn't deal with that kind of thing very well sadly.